med-inventory/src/components/purchase-medicine/purchase-layout/purchaseLayout.css

```diff
@@ -0,0 +1,34 @@
+.card-style{
+    width:100%;
+    height:80vh;
+    margin-top : 10px;
+}
+.preview-button{
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    padding-bottom: 20px;
+}
+.router-outlet-style{
+    flex-grow: 1;
+    padding : 20px;
+}
+.MuiCard-root {
+    overflow: Auto;
+}
+.layout-style{
+    display: flex; 
+    flex-direction: row;
+    justify-content: space-between;
+}
+.grid-box{
+   width: 70vw;
+}
+
```

Comment:
Either leave space between two style definitions or don't leave. Be consistent in whatever approach you are following

Reply:
_I would recommend space between two definition for better readability, especially when we have nested blocks_

Reply:
ok